tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -3128,6 +3147,14 @@ public function getCommentUrlRegExp(): string {
 		return "/{$basePath}remote.php/dav/comments/files/([0-9]+)";
 	}
 
+	/**
+	 *
+	 * @return string
+	 */
+	public static function getGroupUUIDv4Regex(): string {
+		return '[0-9a-f]{8}-[0-9a-f]{4}-4[0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}';
+	}
```

Comment:
can't we use this existing code 
```
	/**
	 *
	 * @return string
	 */
	public static function getUUIDv4Regex(): string {
		return '[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}';
	}
```

Reply:
the value returned by the existing function and the group id does not match because of the case insensitivity.